modules/quick/inventory/src/Plugin/QuickForm/Inventory.php

```diff
@@ -121,7 +123,7 @@ public function defaultConfiguration() {
     return [
       'asset' => NULL,
       'measure' => NULL,
-      'units' => NULL,
+      'units' => '',
```

Comment:
Is there any reason this can't be `NULL` by default?

Reply:
Oh sure it could be. I'll fixup that.